libflux/flux-core/src/semantic/types.rs

```diff
@@ -246,6 +258,7 @@ impl cmp::PartialOrd for Kind {
 
 /// MonoType represents a specific named type.
 #[derive(Debug, Display, Clone, PartialEq, Serialize)]
+#[allow(missing_docs)]
```

Comment:
MonoType represents a Flux type, the type may be unknown represented as a type variable or may be a known concrete type.